source/jormungandr/jormungandr/olympic_site_params_manager.py

```diff
@@ -140,6 +112,14 @@ def get_json_content(self, s3_object):
             logger.exception('Error while loading file: {}'.format(s3_object.key))
             return {}
 
+    def clean_and_get_olympic_site_params(self):
+        result = deepcopy(self.olympic_site_params)
+        for key, value in result.items():
+            for event in value.get("events", []):
+                del event["start"]
+                del event["end"]
+        return result
```

Comment:
don't make deepcopy, and just return self.olympic_site_params with the "start" and "end" added

Reply:
Done